tests/Auth0/Listeners/BlockClientsTest.php

```diff
@@ -12,7 +12,7 @@
 use App\Json;
 use GuzzleHttp\Psr7\Response;
 use PHPUnit\Framework\MockObject\MockObject;
-use PHPUnit\Framework\TestCase;
```

Comment:
Maybe the PHPUnit TestCases can stay as is?

Reply:
In my testing changing it to the custom testcase didn't slow things down. Isn't it less confusing if we only use 1 testcase everywhere?

Reply:
Ok, fine for me!